src/storage/src/hummock/local_version_manager.rs

```diff
@@ -327,37 +333,34 @@ impl LocalVersionManager {
         true
     }
 
-    /// Waits until the local hummock version contains the epoch. We will wait for different epochs
-    /// according to `HummockReadEpoch`'s type
-    pub async fn wait_epoch(&self, wait_epoch: HummockReadEpoch) -> HummockResult<()> {
-        if wait_epoch.get_epoch() == HummockEpoch::MAX {
+    /// Waits until the local hummock version contains the epoch. If `wait_epoch` is `Current`,
+    /// we will only determine that it is greater than `max_current_epoch` and not wait
+    pub async fn try_wait_epoch(&self, wait_epoch: HummockReadEpoch) -> HummockResult<()> {
+        let wait_epoch = match wait_epoch {
+            HummockReadEpoch::Committed(epoch) => epoch,
+            HummockReadEpoch::Current(epoch) => {
+                assert!(
+                    epoch > self.local_version.read().get_max_current_epoch()
```

Comment:
I don't get the assertion. Why is `epoch <= self.local_version.read().get_max_current_epoch()` not allowed?